packages/kit/.gitignore

```diff
@@ -3,9 +3,9 @@
 /assets/*
 /dist
 /docs
-/test/**/.svelte-kit
+.svelte-kit
```

Comment:
I liked the comment you had in the last version. Maybe move this back to the bottom and restore the comment?